config/locales/en.yml

```diff
@@ -20,16 +20,17 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  layout:
```

Comment:
...that's why I didn't use layouts, because this is meant to be shared among themes.